File: camisas-web/src/app/features/camisa/models/camisa.model.ts
export interface Camisa {
  id: number;
  descripcion: string;
  marca: string;
  color: string;
  talla: string;
  manga: string;
  stock: number;
  precioCosto: number;
  precioVenta: number;
  estante: string;
  estado: string;
}

export interface ApiResponseCamisa {
  contenido: Camisa[];
  paginaActual: number;
  tamanio: number;
  totalElementos: number;
  totalPaginas: number;
  primera: boolean;
  ultima: boolean;
  vacia: boolean;
}

// ---- para el evio del form en crear camisa
export interface CamisaCreate {
  descripcion: string;
  id_marca: number;
  color: string;
  talla: 'S' | 'M' | 'L' | 'XL';
  manga: 'Corta' | 'Larga';
  stock: number;
  precio_costo: number;
  precio_venta: number;
  id_estante: number;
  estado: boolean;  
}

export interface Marca   { id_marca: number; descripcion: string; }
export interface Estante { id_estante: number; descripcion: string; }

File: backend/src/main/java/com/cibertec/backend/domain/camisa/model/CamisaModel.java
package com.cibertec.backend.domain.camisa.model;

import lombok.Data;

import java.math.BigDecimal;

@Data
public class CamisaModel {
    private Long camisaId;
    private String descripcion;
    private Long marcaId;
    private String marca;
    private String color;
    private String talla;
    private String manga;
    private Integer stock;
    private BigDecimal precioCosto;
    private BigDecimal precioVenta;
    private Long estanteId;
    private String estante;
    private Boolean estado;
}
